target/linux/ar71xx/files/arch/mips/ath79/mach-rb2011.c

```diff
@@ -94,7 +94,7 @@ static struct mtd_partition rb2011_nand_partitions[] = {
 		.size	= (4 * 1024 * 1024) - (256 * 1024),
```

Comment:
the description of the patch explains why it is there and not what it does. please add a more explanatory description